draft-ietf-moq-requirements.md

```diff
@@ -675,6 +680,11 @@ groups and just wait for the next group when needed.
 
 Of course, having longer blocks create other issues. Realtime conferences also need to accomodate the occasional occasional late comer, or the disconnected user who want to resynchronize after a network event. This drives a need for synchronization "between access points". For example, rather than waiting for 30 seconds before connecting, the user might quickly download the "key" frames of the past 30 seconds and replay them in order to "synchronize" the video decoder.
 
+### Media Insertion and Redirection
+
+In all use cases it may be necessary for consumers to switch sources based on either triggering provided within a stream of media, or externally in particular to support the insertion of advertising or promotions, but may be useful for other requirements where an upstream encoder or producer is unable to do. Relays may also require consumers to relocate their subscriptions of media in order to facilitate this.
```

Comment:
This is still a draft PR, but I'm finding the first sentence a bit difficult to parse in one or two passes. Is it saying this?

> In all use cases it may be necessary for consumers to switch sources based on either triggering provided within a stream of media, or externally in particular to support the insertion of advertising or **promotions. The capability to switch sources may** be useful for other requirements where an upstream encoder or producer is unable to do.